storage/innobase/srv/srv0start.cc

```diff
@@ -1305,7 +1334,31 @@ dberr_t srv_start(bool create_new_db)
 		}
 
 		create_new_log = srv_log_file_size == 0;
-		if (!create_new_log) {
+		if (create_new_log) {
+			if (flushed_lsn < lsn_t(1000)) {
```

Comment:
FWIW, the minimum LSN before is 8192 or 8192+12=8204. I think that we could simply check for zero here. In 685d958e38b825ad9829be311f26729cccf37c46 [MDEV-14425](https://jira.mariadb.org/browse/MDEV-14425) it was changed to `log_sys.FIRST_LSN=12288` and this code was removed. Did you also test a 10.11 version of this change?

Reply:
1. I do not intend to include any other functional changes part of this patch. The numbers and code logic remains exactly same. If we intend to backport MDEV-14425 to 10.5, it is better to open and MDEV.

2. This part of the patch (Issue-1:  in description) won't be required from 10.6 onwards. I have checked in 10.6 but not in 10.11.